src/plugins/content/accessibility/jumplinks/shortcuts.js

```diff
@@ -73,6 +73,7 @@ const defaults = {
 export default function shortcutsBoxFactory(config) {
     const shortcutsBox = component({}, defaults)
         .on('render', function () {
+            const ESK_KEY_CODE = 27;
```

Comment:
I  suggest to move it outside of the render event, since the value is independent of it.